mysql-8.0/Dockerfile

```diff
@@ -1,34 +1,34 @@
-FROM mysql:8.0.30-debian
+FROM mysql:8.0.32-oracle
 MAINTAINER groonga
 
-ENV groonga_version=12.0.6 \
-    mroonga_version=12.06
+ENV groonga_version=13.0.1 \
+    mroonga_version=13.01 \
 
 RUN mkdir -p /etc/mysql/mysql.conf.d && \
     touch /etc/mysql/mysql.conf.d/default-auth-override.cnf && \
     touch /etc/mysql/mysql.conf.d/lowercase-table-names.cnf && \
-    apt update && \
-    apt install -y -V --no-install-recommends \
-      ca-certificates \
-      lsb-release \
+    major_version=$(cut -d: -f5 /etc/system-release-cpe | grep -o "^[0-9]") && \
+    microdnf install -y \
       wget && \
-    code_name=$(lsb_release --codename --short) && \
-    wget https://packages.groonga.org/debian/groonga-apt-source-latest-${code_name}.deb && \
-    apt install -y -V --no-install-recommends \
-      ./groonga-apt-source-latest-${code_name}.deb && \
-    rm groonga-apt-source-latest-${code_name}.deb && \
-    apt update && \
-    apt install -y -V --no-install-recommends \
-      groonga-bin=${groonga_version}-1 \
-      groonga-normalizer-mysql \
-      groonga-tokenizer-mecab=${groonga_version}-1 \
-      mysql-community-8.0-mroonga=${mroonga_version}-1 && \
-    sed \
-      -i'' \
-      -e 's,@MRN_DATA_DIR@,/usr/share/mroonga,g' \
-      /usr/share/mroonga/install.sql && \
-    apt clean && \
-    rm -rf /var/lib/apt/lists/*
+    microdnf install -y \
+      cmake-filesystem \
+      lz4 \
+      snappy \
+      xxhash-libs && \
+    epel_base=https://dl.fedoraproject.org/pub/epel/${major_version}/Everything/x86_64/Packages && \
+    wget ${epel_base}/m/msgpack-3.1.0-3.el8.x86_64.rpm && \
+    wget ${epel_base}/r/re2-20190801-1.el8.x86_64.rpm && \
+    powertools_base=https://repo.almalinux.org/almalinux/${major_version}/PowerTools/x86_64/os/Packages && \
+    wget ${powertools_base}/glog-0.3.5-3.el8.x86_64.rpm && \
+    wget ${powertools_base}/gflags-devel-2.1.2-6.el8.x86_64.rpm && \
+    wget ${powertools_base}/gflags-2.1.2-6.el8.x86_64.rpm && \
+    apache_arrow_base=https://apache.jfrog.io/artifactory/arrow/almalinux/${major_version}/x86_64/Packages && \
+    wget ${apache_arrow_base}/arrow11-libs-11.0.0-1.el8.x86_64.rpm && \
```

Comment:
It seems that we can use the following:

```bash
wget https://apache.jfrog.io/artifactory/arrow/almalinux/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm
rpm -Uvh apache-arrow-release-latest.rpm
```

But the following warning is reported:

```text
(microdnf:528): libdnf-WARNING **: 02:23:25.359: failed to parse public key for /etc/pki/rpm-gpg/RPM-GPG-KEY-Apache-Arrow
```

I don't know why but `sed -i'' -e '26d' /etc/pki/rpm-gpg/RPM-GPG-KEY-Apache-Arrow` is a workaround.